mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
Ultimately, what I want to avoid is getting 500 Internal Server Errors based on something the client can do. E.g. `curl http://example.com/oidc/logout/`. So we need something to deal with this view being executed by anonymous users.